packages/solid-start-server/vite.config.ts

```diff
@@ -6,7 +6,10 @@ import minifyScriptPlugin from './vite-minify-plugin'
 import type { ViteUserConfig } from 'vitest/config'
 
 const config = defineConfig({
-  plugins: [minifyScriptPlugin(), solid()] as ViteUserConfig['plugins'],
+  plugins: [
+    minifyScriptPlugin(),
+    solid({ solid: { generate: 'ssr' } }),
```

Comment:
is this needed?

Reply:
Without this the JSX transform emits code for hydrating on client instead of rendering HTML on server. On normal Vite apps we use `vite build --ssr` to configure the JSX transform but it seems like it doesn't work well with Vite Library Mode